src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 200) {
```

Comment:
The constructor of the `Animal` class should not have a default health value of 200. The task specifies that the default health should be 100.